rs/interfaces/src/p2p/consensus.rs

```diff
@@ -145,3 +124,26 @@ pub trait ArtifactAssembler<A1: IdentifiableArtifact, A2: PbArtifact>:
         peers: P,
     ) -> impl std::future::Future<Output = Result<(A1, NodeId), Aborted>> + Send;
 }
+
+/// The artifact filter function returns a value that defines 3 possible handling logics when
+/// an artifact or ID is received.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum FilterValue {
+    /// The client doesn't need the corresponding artifact for making progress so it can safely be dropped.
+    Unwanted,
+    /// The client may need later the artifact.
+    MaybeWantsLater,
+    /// The artifact needs to be delivered to the client.
+    Wants,
```

Comment:
Shouldn't this be changed for consistency?
```suggestion
    MaybeWantedLater,
    /// The artifact needs to be delivered to the client.
    Wanted,
```

Reply:
I would prefer the verbs to be in present tense. Unwanted in this case an adjective so I don't think it sounds so much worse. I think I would prefer to keep it as is.

Reply:
I see. But `Wanted` is also an adjective just like `Unwanted`?